frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -144,6 +146,35 @@ function ReaderCoptListener:updatePageInfoOverride(pageno)
     if self.reading_percent == 1 then
         page_info = page_info .. percentage_pre .. percentage_fmt:format(percentage*100) .. percentage_post
     end
+
+    if self.battery == 1 and self.battery_percent == 1 then -- append battery percentage
+        local batt_pre = "["
+        local batt_post = "]"
+        local batt_val = ""
```

Comment:
Better have `batt_val = nil`
For devices without battery, it would show as ![image](https://github.com/koreader/koreader/assets/24273478/644d8e53-abe6-4b39-b00c-8b319016e1df)